src/hooks/useFarmStats.ts

```diff
@@ -1,19 +1,19 @@
-import { useEffect, useState } from 'react'
+import { useEffect, useState } from 'react';
 
-import { BigIntish } from './types'
+import { BigIntish, TimeRemaining } from 'util/types';
 
-import gql from 'graphql-tag'
-import { Client } from 'urql'
+import dayjs from 'dayjs';
+import gql from 'graphql-tag';
+import { Client } from 'urql';
 
-import { WPOKT_SUBGRAPH_URL } from 'util/constants'
+import { getTimeRemaining } from 'util/helpers';
+import { WPOKT_SUBGRAPH_URL } from 'util/constants';
 
-const RETRY_EVERY = 3000
-const DAYS_IN_ONE_MONTH = 30
-const ONE_DAY = 86400000
-const ZERO_BIG_INT = BigInt(0)
+const RETRY_EVERY = 3000;
+const DAYS_IN_ONE_MONTH = 30;
+const ZERO_BIG_INT = BigInt(0);
 
-const graphqlClient = new Client({ url: WPOKT_SUBGRAPH_URL || 
-  'Your farm subgraph URL' })
+const graphqlClient = new Client({ url: WPOKT_SUBGRAPH_URL || 'Your farm subgraph URL' });
```

Comment:
Also nitpicking:
```suggestion
const graphqlClient = new Client({ url: WPOKT_SUBGRAPH_URL });
```

If someone's not passing a subgraph URL, they should definitely do so instead of us having a random string. :)